sourcegraph/cody
# Edit: Fix JetBrains issue with range expansion

## Description

Uses `document.lineAt` to get the end of a line, rather than `Number.MAX_SAFE_INTEGER`.

## Test plan

1. Create edits in JetBrains
2. Undo them
3. Check the undo functions correctly (only original code is returned)
